lgc/interface/lgc/Pipeline.h

```diff
@@ -164,6 +164,15 @@ struct ShaderOptions {
 
   /// Default unroll threshold for LLVM.
   unsigned unrollThreshold;
+
+  /// Unroll loops by specified amount. 0 is default, 1 is no unroll.
```

Comment:
Please change /// to //. I decide to clean up all unnecessary style of "///". Some are introduced by LGC separation which extract certain interface definitions from llpc.h.

Reply:
Done